SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -373,6 +373,43 @@ void USpatialSender::ProcessUpdatesQueuedUntilAuthority(Worker_EntityId EntityId
 	}
 }
 
+void USpatialSender::FlushPackedUnreliableRPCs()
+{
+	if (UnreliableRPCs.Num() == 0)
+	{
+		return;
+	}
+
+	check(NetDriver->GetWorld() != nullptr);
```

Comment:
Why check for this here?

Reply:
Ah, forgot to remove this from when this code was getting the entity id from the local player controller (which it got from the `NetDriver->GetWorld()->GetGameInstance()`)